apps/tutorial_servo_control/src/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include "mbed.h"
+#include "TutorialServo.h"
+#include "CANMsg.h"
+
+
+CAN can(CAN1_RX, CAN1_TX);
+
+int main() {
+
+        TutorialServo servo(PA_1);
+        CANMessage msg;
+
+        while (true) {
+                if(can.read(msg)){
+
+                        printf("Message received: %d\n", msg.data[0]);
```

Comment:
can messages have a getPayload() function that we use to extract the data from a msg. use that instead